File: ds_&_algorithms/algorithm_analysis/constant_time_complexity/java/ConsTimeComp.java
package algorithm_analysis.constant_time_complexity.java;

public class ConsTimeComp {

    static int sum(int x, int y){
        int result = x+y; // Constant time operation
        return result;
    }

    public static void main(String[] args){
        int adder = 10;
        int addee = 10;
        int finalsSum = sum(addee, adder);
        System.out.println("The value of sum is:" +finalsSum);
    }
    
}

File: ds_&_algorithms/algorithm_analysis/constant_time_complexity/go/ConsTimeComp.go
package main

import "fmt"

func sum(x int, y int) int {
	var result int = x + y // Constant time operation

	return result
}

func main() {
	var adder int = 10
	var addee int = 10

	var finalSum int = addee + adder

	fmt.Printf("The value of sum is %v", finalSum)
}
